frontend/pages/DashboardPage/cards/ActivityFeed/components/ActivityAutomationDetailsModal/_styles.scss

```diff
@@ -0,0 +1,19 @@
+.activity-automation-details-modal {
+  box-sizing: border-box;
+  width: 811px;
```

Comment:
I've seen this pattern, I'm wondering if there's a reason we're still manually setting this and not using Modal `width` prop

Reply:
Not sure. I based this modal on the script execution result modal as that seemed closest.